dwave/cloud/api/client.py

```diff
@@ -801,14 +817,14 @@ class DefaultSession(VersionedAPISessionMixin,
         'version_strict_mode': True,
 
         # enable conditional requests and response caching, see :class:`CachingSession`
-        'cache': dict(enabled=False),       # type: CachingSession.CacheConfig
+        'cache': dict(enabled=False),       # type: CachingSession.ExtendedCacheConfig
 
         # api deprecation message handling, see :class:`DeprecationAwareSessionMixin`
         'on_deprecation': dict(log=True, warn=True, store=True),
     }
 
     # client instance config, populated on init from kwargs overriding DEFAULTS
-    config = None
+    config: dict
```

Comment:
Just confirming that there's no case where `DWaveAPIClient.config` might be called (on the class), expecting a `None` to return? With this change an `AttributeError` would be raised I believe.

Reply:
Nope, that's why I simplified it here.